signaling-server/signaling_server.py

```diff
@@ -213,7 +233,7 @@ def _isAuthorized(user_password):
 
     # Look for ice servers file
     if args.ice_servers is not None:
-        with open(args.ice_servers) as file:
+        with args.ice_servers.open() as file:
```

Comment:
`args.ice_servers` is now a `pathlib.Path` object, which can be opened using a method